editor/import/resource_importer_scene.cpp

```diff
@@ -2422,32 +2422,36 @@ Error ResourceImporterScene::import(const String &p_source_file, const String &p
 	_post_fix_animations(scene, scene, node_data, animation_data, fps);
 
 	String root_type = p_options["nodes/root_type"];
-	root_type = root_type.split(" ")[0]; // full root_type is "ClassName (filename.gd)" for a script global class.
-
-	Ref<Script> root_script = nullptr;
-	if (ScriptServer::is_global_class(root_type)) {
-		root_script = ResourceLoader::load(ScriptServer::get_global_class_path(root_type));
-		root_type = ScriptServer::get_global_class_base(root_type);
-	}
-
-	if (root_type != "Node3D") {
-		Node *base_node = Object::cast_to<Node>(ClassDB::instantiate(root_type));
-
-		if (base_node) {
-			scene->replace_by(base_node);
-			scene->set_owner(nullptr);
-			memdelete(scene);
-			scene = base_node;
-		}
-	}
-
-	if (root_script.is_valid()) {
-		scene->set_script(Variant(root_script));
-	}
-
-	if (p_options["nodes/root_name"] != "Scene Root") {
-		scene->set_name(p_options["nodes/root_name"]);
-	} else {
+	if (!root_type.is_empty()) {
+		root_type = root_type.split(" ")[0]; // full root_type is "ClassName (filename.gd)" for a script global class.
```

Comment:
```suggestion
		root_type = root_type.split(" ")[0]; // Full root_type is "ClassName (filename.gd)" for a script global class.
```
Code style nitpick

Reply:
Pre-existing code, but I agree this is a good improvement to make now.